### PR TITLE
for feature deep delete add data provider and child condition configu…

### PR DESCRIPTION
**Pull request by baumannsven**
Note that when you merge this the other translated attributes and the change to the core must also be taken over!

### CODE REVIEW

File: travis.php.ini [hunk 1]
```diff
@@ -0,0 +1 @@
+memory_limit = 2G
```
Comment (discordier) [inline travis.php.ini]: Is this really needed?

If so, we must add it to `.gitattributes` and also remove the `minimum-stability` from `composer.json`

**Comment by baumannsven** [comment]
Yes the memory limit is needed. You can see by build history from this pull request. By PHP Version < 7 we must have over 1GB memory.

**Comment by baumannsven** [comment]
I have remove the `minimum-stabilty` from `composer.json`. After gives a problem with delete the stabilty. 

You can see here:
https://travis-ci.org/MetaModels/attribute_translatedcheckbox/jobs/182152116

**Comment by discordier** [comment]
Merged in 6bad98d259ce0f812f8e4d69e625370b9df32204

**Pull request closed without merging**